tests/test_ble_lifecycle_receive_targets.py

```
@@ -665,259 +727,652 @@ def join(timeout: float | None = None) -> None:
+            BLEReceiveRecoveryService,
+            "_coordinator_check_and_clear_event",
+            staticmethod(lambda *_args, **_kwargs: True),
+        )
+        proceed, poll_without_notify = BLEReceiveRecoveryService._wait_for_read_trigger(
             iface,
             coordinator=SimpleNamespace(),
             wait_timeout=0.1,
         )
-        is False
-    )
+        assert proceed and poll_without_notify
```

Ruff (0.15.6)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)